modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,3 +64,63 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+[discrete]
+== Redfish APIs
+
+Several redfish API endpoints are called onto your BCM when using the bare-metal installer-provisioned infastructure.
+
+[IMPORTANT]
+====
+You need to ensure that your redfish APIs successfully operate prior to installation.
```

Comment:
@kelbrown20 Lets change this to:
`You need to ensure that your BMC supports all of the redfish APIs prior to installation`